src/customize-pop/application-settings.md

```diff
@@ -0,0 +1,63 @@
+| Integration | Disable or enable system-wide features used by an application. These features include the ability to appear in search, display notification, and running in the background. |
+| Default Handlers | This determines which file types are associated with an application. Click an entry to display more information about the file types assigned to the application. Multiple programs can be associated with the same file types. See the [Default Applications](/customize-pop/application-settings.md#default-applications) to specify an application that should always open a certain media type. |
+| Usage | Display an application's disk usage. Click on the `Storage` button to view space used by the application itself, data created by the application, and temporary data cached by the application. Click `Clear Cache` to clear an application's locally stored data.
+
+## Notifications
+
+Settings to control application notification behavior are located in the `Settings` ➞ `Notifications` tab.
+
+![Notifications Settings](/images/application-settings/notifications-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Do Not Disturb | Disable all application notifications globally. |
+| Lock Screen Notifications | Enable setting toggling for the `Lock Screen Notifications` and `Show Message Content on Lock Screen` options. |
+
+Application notifications appear in the upper center of the screen. Click the date and time to view recent notifications.
+
+![Recent Notifications](/images/application-settings/recent-notifications.png)
+
+### Configure Notification Behavior for a Single Application
+
+Select an application from the list to configure its notification behavior.
+
+![Single App Options](/images/application-settings/single-app-options.png)
+
+| Option | Function |
+|--------|----------|
+| Notifications | Toggle an application's notifications on or off. |
+| Sound Alerts | Toggle sound alerts that will accompany notifications for this application. |
+| Notification Popups | Enable notification popups on the desktop; notifications will continue to appear in the notification list. |
+| Show Message Content in Popups | Include the notification's written content in the popup. |
+| Lock Screen Notifications | Display application notifications at the login screen when the computer is locked. These settings cannot be enabled unless the Lock Screen Notifications setting is enabled. |
+| Show Message Content on Lock Screen | Displays an application's written notification content in the login screen when the computer is locked. |
+
+## Default Applications
+
+Options to choose default applications for Web browsing, Email, Calendars, Music, Videos, and Photos are located in `Settings` ➞ `Default Applications`.
```

Comment:
"Web browsing" only having its first word capitalized reads a little weird; I understand that you wanted to match the capitalized "Web" in the screenshot, but you also used "Email" instead of "Mail." You could get around this by being more descriptive, like:

> Options to set the default web browser, email client, calendar app, music player, video player, and photo viewer are located in `Settings` ➞ `Default Applications`.